plugins/@grouparoo/mailchimp/src/lib/shared/updateProfile.ts

```diff
@@ -17,6 +22,7 @@ export interface UpdateProfileMethod {
 }
 
 export const updateProfile: UpdateProfileMethod = async ({
+  syncOperations,
```

Comment:
I think because we're passing `syncOperations` here we can remove `noDelete` and `noCreate`. `syncOperations` should be enough to determine what we should do.